src/components/fields/select/select.tsx

```diff
@@ -1,21 +1,22 @@
-import { Form, InputSelect } from "@lifesg/react-design-system";
-import React, { useEffect, useState } from "react";
+import { Form } from "@lifesg/react-design-system/form";
+import { InputSelect } from "@lifesg/react-design-system/input-select";
+import { useEffect, useState } from "react";
 import * as Yup from "yup";
 import { TestHelper } from "../../../utils";
 import { useValidationSchema } from "../../../utils/hooks";
 import { IGenericFieldProps } from "../../frontend-engine";
-import { ISelectRef, ISelectSchema } from "./types";
+import { ISelectSchema } from "./types";
 
-export const Select = React.forwardRef<ISelectRef, IGenericFieldProps<ISelectSchema>>((props, ref) => {
```

Comment:
by default react-hook-form wants to get the ref of each field to interact with it (e.g. through onChange, onBlur, etc), however in our case, not all of our fields are able to apply the ref (e.g. image upload in the future), hence we preemptively avoided this approach and stick to apply the events manually